tests/utils.py

```diff
@@ -125,34 +132,99 @@ def clean_layout(layout):
 def output_layout(layout, depth=1):
     if 'tag' in layout:
         return ('  ' * depth
-            + '* {tag}{n[content][size][0]}x{n[content][size][1]}'
-              ' @ ({n[content][position][0]}, {n[content][position][1]})'
-              '\n'.format(
-                    n=layout,
-                    tag=('<' + layout['tag'] + '> ') if 'tag' in layout else '',
-                    # text=(": '" + layout['text'] + "'") if 'text' in layout else ''
-                )
-            # + '  ' * depth
-            # + '  padding: {n[padding_box][size][0]}x{n[padding_box][size][1]}'
-            #   ' @ ({n[padding_box][position][0]}, {n[padding_box][position][1]})'
-            #   '\n'.format(n=layout)
-            # + '  ' * depth
-            # + '  border: {n[border_box][size][0]}x{n[border_box][size][1]}'
-            #   ' @ ({n[border_box][position][0]}, {n[border_box][position][1]})'
-            #   '\n'.format(n=layout)
-            + ''.join(
-                    output_layout(child, depth=depth + 1)
-                    for child in layout.get('children', [])
-                ) if layout else ''
-            + ('\n' if layout and layout.get('children', None) and depth > 1 else '')
-        )
+                + '* {tag}{n[content][size][0]}x{n[content][size][1]}'
+                ' @ ({n[content][position][0]}, {n[content][position][1]})'
+                '\n'.format(
+                        n=layout,
+                        tag=('<' + layout['tag'] + '> ') if 'tag' in layout else '',
+                        # text=(": '" + layout['text'] + "'") if 'text' in layout else ''
+                    )
+                # + '  ' * depth
+                # + '  padding: {n[padding_box][size][0]}x{n[padding_box][size][1]}'
+                #   ' @ ({n[padding_box][position][0]}, {n[padding_box][position][1]})'
+                #   '\n'.format(n=layout)
+                # + '  ' * depth
+                # + '  border: {n[border_box][size][0]}x{n[border_box][size][1]}'
+                #   ' @ ({n[border_box][position][0]}, {n[border_box][position][1]})'
+                #   '\n'.format(n=layout)
+                + ''.join(
+                        output_layout(child, depth=depth + 1)
+                        for child in layout.get('children', [])
+                    ) if layout else ''
+                + ('\n' if layout and layout.get('children', None) and depth > 1 else ''))
     else:
-        return ('  ' * depth
-            + "* '{text}'\n".format(text=layout['text'].strip())
-        )
+        return ('  ' * depth + "* '{text}'\n".format(text=layout['text'].strip()))
+
+
+def copy_fonts(system=False):
```

Comment:
Although it does do some copying, this is more of an "install" fonts - you don't have any control over the destination.

Reply:
True, will rename!